bin/typhon_env.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr//bin/env bash
+TYPHON_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Comment:
Is this the normal way to do this? Seems weird to cd and pwd to get the full path

Reply:
I think we usually do `readlink`, but I wanted this to work on MacOS as well (which doesn't ship with `readlink`)

Reply:
is `dirname` not sufficient?

Reply:
I like this... my only concern is that the result of the script might change based on directory. Let's say you have `typhon` installed in your environment, but you are testing a modified local checkout. Maybe I'm overthinking this.

Reply:
I like yours better, slightly more flexible. Unfortunately the people who really care about the designer are often the least knowledgable... trying to make it as simple as possible.